templates/core/terraform/resource_processor/vmss_porter/cloud-config.yaml

```diff
@@ -39,6 +39,7 @@ write_files:
       AZURE_TENANT_ID=${arm_tenant_id}
       ARM_USE_MSI=true
       APPLICATIONINSIGHTS_CONNECTION_STRING=${app_insights_connection_string}
+      NUMBER_PROCESSES=${resource_processor_number_processes_per_instance}
```

Comment:
Very very very nit: I think it's a bit more readable when "number" goes to the end of the name, e.g. "PROCESSES_NUMBER" (similar how units are usually put at the end)